workers/notifier/src/main/kotlin/notifier/NotifierOrtResultGenerator.kt

```diff
@@ -137,4 +143,8 @@ internal class NotifierOrtResultGenerator(
 
         return jobStatusLabels
     }
+
+    private fun getRunId(ortRun: OrtRun): Map<String, String> {
```

Comment:
Better name this `getOrtRunLabels()` to align with the other functions.

Reply:
Function no longer needed after latest refactoring of PR.